wob.c

```diff
@@ -274,10 +274,27 @@ main(int argc, char **argv)
 		return EXIT_FAILURE;
 	}
 
-	uint32_t *argb = wob_create_argb_buffer(&app), i, k;
+
+	// Parse arguments
+	int c;
+	int timeout_msec = 1000;
+	while ((c = getopt(argc, argv, "t:")) != -1) {
+		switch (c){
+			case 't':
+				timeout_msec = atoi(optarg);
+				if (timeout_msec < 0){
+					fprintf(stderr, "Timeout must be a positive value.");
+					return EXIT_FAILURE;
+				}
+				break;
+		}
+	}
+
+	uint32_t * argb = wob_create_argb_buffer(&app), i, k;
```

Comment:
This got added while resolving merge conflicts. No worries, it's supposed to be there anyway,